keyboards/q4z/info.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file should be filled out or removed. 

Ideally, filled out.  If you have the kbfirmware json, you can use this: https://noroadsleft.github.io/kbf_qmk_converter/